packages/bootstrap/extra/dcos_internal_utils/utils.py

```diff
@@ -18,6 +21,24 @@
 dcos_etc_path = install_path / 'etc'
 
 
+def get_user_config() -> Dict[str, str]:
+    """
+    Returns the contents of the cluster `config.yaml` file as a dictionary.
```

Comment:
```suggestion
    Returns the contents of the cluster `user.config.yaml` file as a dictionary.
```

Reply:
It's the `config.yaml` file on the bootstrap node and in the docs. It gets put into the `user.config.yaml` file on the cluster nodes. So I'd probably clarify that rather than change it, but I'll leave it for now.